query-engine/connector-test-kit-rs/test-configs/planetscale-wasm

```diff
@@ -0,0 +1,9 @@
+{
+    "connector": "vitess",
+    "version": "planetscale.js",
+    "driver_adapter": "planetscale",
+    "driver_adapter_config": {
+        "proxy_url": "http://root:root@127.0.0.1:8085"
+    },
+    "external_test_executor": "Wasm"
+}
```

Comment:
I'm a bit lost with the difference between the `connector`, the `version` and the `driver_adapter` field. Could you elaborate how they differ and why we need all three?

Reply:
Good catch! That's true. I left both separated only because neon has two potential driver adapters (HTTP and WS). But I can definitely derive the driver adapter from the connector :+1:,  because in the end, if we test neon-http, we can create a different connector version.  For now they are, as you said, tightly coupled.